File: EventQRBackend/eventqr/src/main/java/com/thedavelopers/eventqr/features/registrations/model/dto/RegistrationSubmissionResponse.java
package com.thedavelopers.eventqr.features.registrations.model.dto;

import com.thedavelopers.eventqr.shared.port.QrCredentialPort.QrCredentialSnapshot;

public record RegistrationSubmissionResponse(RegistrationResponse registration, QrCredentialSnapshot qrCredential) {
}

File: EventQRMobile/app/src/main/java/com/thedavelopers/eventqr/features/registrations/model/dto/RegistrationSubmissionResponse.kt
package com.thedavelopers.eventqr.features.registrations.model.dto

import com.thedavelopers.eventqr.features.qrcredential.model.dto.QrCredentialSnapshot

data class RegistrationSubmissionResponse(
    val registration: RegistrationResponse,
    val qrCredential: QrCredentialSnapshot,
)